napari/_qt/qt_viewer.py

```diff
@@ -168,10 +168,8 @@ def __init__(self, viewer):
         self._last_visited_dir = str(Path.home())
 
         self._cursors = {
-            'disabled': QCursor(
-                QPixmap(':/icons/cursor/cursor_disabled.png').scaled(20, 20)
-            ),
             'cross': Qt.CrossCursor,
+            'disabled': Qt.ForbiddenCursor,
```

Comment:
I would completely drop `disabled` as I don't think we use it at all, no need to keep it around